app/models/proxy.rb

```diff
@@ -327,18 +327,6 @@ def track_apicast_version_change
     end
   end
 
-  def save_and_async_deploy(attrs, user)
```

Comment:
Isn't this used somewhere else?

Reply:
```
🐚 rg save_and_async_deploy
```
shows nothing

Reply:
It was used here: https://github.com/3scale/porta/pull/2487/files#diff-8aa693f2978224422216c485c5a77372dd282ca5599f558c185744473e8a10d7L130 
which is already removed. 

Thus this PR also looks good.

Reply:
I think it is more of a problem to remove usages and leave implementation because it may not show up in tests. While keeping calls but remove implementation is more likely to break :)